frontend/src/components/Playback/Playback.js

```diff
@@ -75,33 +76,66 @@ const Playback = ({
         } else {
             finishedPlaying();
         }
-
     }, []);
 
     // Play audio
     const playAudio = useCallback(
         (index) => {
-            // Only initialize webaudio if section is local            
-            let latency = 0;      
-            // Store player index
-            setPlayerIndex(index);
-            // Determine if audio should be played
-            if (playConfig.mute) {
-                setPlayerIndex(-1);
-                audio.pause();
-                return;
-            }
-            // Volume 1
-            audio.setVolume(1);
+            let latency = 0;
+            if (playConfig.play_method === 'BUFFER' && !playConfig.external_audio) {
+                console.log('Play buffer (local)');                
 
-            // Cancel active events
-            cancelAudioListeners();
+                // Determine latency for current audio device
+                latency = webAudio.getTotalLatency();
+
+                // Store player index
+                setPlayerIndex(index);
```

Comment:
what is the `playerIndex` used for? I think we need to clean something up here -- not necessarily in this branch though -- because there is a `prevPlayerIndex` and `lastPlayerIndex` ref in this component, and it's not documented what it's used for.